cdap-watchdog/src/main/java/co/cask/cdap/logging/write/LogCleanup.java

```diff
@@ -109,13 +121,127 @@ public Void call() throws Exception {
     }
   }
 
+  private void cleanFilesWithoutMeta(final long tillTime, final Map<String, NamespaceId> namespacedLogBaseDirMap,
+                                     final SetMultimap<String, Location> parentDirs) throws Exception {
+    LOG.info("Starting deletion of log files older than {} without metadata", tillTime);
+    List<NamespaceMeta> namespaceMetaList = namespaceQueryAdmin.list();
```

Comment:
This is kind of a system operation I think. After every x days cdap cleanup logs. I don't think it runs as any specific user. @poornachandra : Thoughts ?